lib/http.js

```diff
@@ -42,6 +42,9 @@ function request(options, cb) {
 }
 
 function get(options, cb) {
+  if (options.method) {
+    options.method = 'GET'
+  }
```

Comment:
Instead of manipulating the options object or silently accepting malformed input it would be better to throw an error.

So please change this to e.g.:

```js
if (options.method != null) {
  if (typeof options.method !== 'string')
    throw new ERR_INVALID_ARG_TYPE('options.method', 'string', options.method);
  if (options.method.toUpperCase() !== 'GET') {
    throw new ERR_INVALID_ARG_VALUE('options.method', options.method,
                                    'is invalid. Only "GET" may be used');
  }
}
```

Reply:
Well, I'll change it.